src/oc_erchef/raml-docs/Makefile

```diff
@@ -12,7 +12,7 @@ else
 		echo "You don't have raml2html, but we'll assume you want it" ; \
```

Comment:
I think the test in this if is wrong. I never got this error message and rather got a command not found error on line 15.

The problem can be solved either by using the `-z` test rather than `-f` or by putting quotes around the variable before passing it to the -f test.